src/components/secondPassphrase/firstStep.js

```diff
@@ -14,13 +14,19 @@ const FirstStep = ({
     </header>
     <Trans>
       <p className={styles.info}>
+
+
         After registration, your second passphrase will be
         {' '}
         <strong>required </strong>
+
+
         when
         {' '}
         <strong>confirming every transaction</strong>
         {' '}
+
+
```

Comment:
I probably don't understand the purpose but looks like there is a lot of empty spaces here, should be like that or not?